### PR TITLE
rerender with conda smithy 3.2.10

### PR DESCRIPTION
**Pull request by notestaff**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `1000` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by notestaff** [comment]
@conda-forge/core @conda-forge/ghc @conda-forge/ghc-bootstrap @conda-forge/cabal @conda-forge/stack @conda-forge/git-annex @conda-forge/datalad  I'd be grateful for any help fixing this rerender of git-annex.   Something changed in conda-forge, that is breaking recipes that have worked before.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ source:
     - 0001-add-magic-and-sandi.patch
 
 build:
-  number: 1
+  number: 1000
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: This isn't needed anymore, you can go back to 0

**Comment by isuruf** [comment]
A workaround would be to do `export LIBRARY_PATH=$PREFIX/lib` before building,

**Comment by notestaff** [comment]
@isuruf that's already being done: https://github.com/conda-forge/git-annex-feedstock/blob/115748322e117f19d8a188d30aaa8f06bc3d7f7e/recipe/build.sh#L11

**Comment by notestaff** [comment]
@eggzilla Do you know of examples of other stack-based Haskell project successfully built with the new conda-forge?

@conda-forge/core @conda-forge/ghc @conda-forge/ghc-bootstrap @conda-forge/cabal @conda-forge/stack @conda-forge/git-annex @conda-forge/datalad If anyone who understands both the conda-forge build process and Haskell/ghc/stack/cabal can help with making git-annex feedstock work with the current conda, this would be much appreciated.

**Pull request closed without merging**